tsfile/src/main/java/org/apache/iotdb/tsfile/file/metadata/enums/TSEncoding.java

```diff
@@ -51,6 +51,35 @@ public static TSEncoding deserialize(short i) {
     }
   }
 
+  /**
+   * give an byte to return a encoding type.
+   *
+   * @param i byte number
+   * @return encoding type
+   */
+  public static TSEncoding byteToEnum(byte i) {
+    switch (i) {

```

Comment:
better to check the size of this parameter as TSDataType

Reply:
Copy that. I'll change it